docker-setup/snowflake-kafka-docker/kafka-connect.yml

```diff
@@ -1,201 +0,0 @@
----
-version: '2'
```

Comment:
why we're removing these template now?

Reply:
oops this is a git ignore mistake, ill add it back it